eclipsePlugin-junit/build.gradle

```
@@ -1,11 +1,33 @@
 apply from: "$rootDir/gradle/checkstyle.gradle"
 apply from: "$rootDir/gradle/jacoco.gradle"
 
+def localProps = new Properties()
+localProps.load(new FileInputStream("$rootDir/eclipsePlugin/local.properties"))
+
+def eclipseRootDir = new File(localProps.getProperty('eclipseRoot.dir'))
```

Can we use just `file(...)` instead of `new File(...)`?

```suggestion
def eclipseRootDir = file(localProps.getProperty('eclipseRoot.dir'))
```